spec/mailers/distribution_mailer_spec.rb

```diff
@@ -1,38 +1,42 @@
-RSpec.describe DistributionMailer, type: :mailer do
+RSpec.describe DistributionMailer, type: :mailer, seed_items: false do
```

Comment:
Just noticing this - `seed_items` actually controls whether we seed _all_ the base data, not just items. Probably should rename this in a future PR.

Reply:
@dorner Can that be a Good First Issue?